evm_arithmetization/src/cpu/kernel/asm/main.asm

```diff
@@ -213,8 +213,10 @@ global check_state_trie:
     %mload_global_metadata(@GLOBAL_METADATA_STATE_TRIE_DIGEST_BEFORE)
     %assert_eq
     // Check that the stored trie data length is correct.
-    %mload_global_metadata(@GLOBAL_METADATA_TRIE_DATA_SIZE) 
-    %assert_eq
+    // TODO: uncommen the following 2 lines
+    //%mload_global_metadata(@GLOBAL_METADATA_TRIE_DATA_SIZE) 
+    //%assert_eq
+    POP // remove
```

Comment:
Is this fixed? As the check is not performed at the moment

Reply:
Ops, I forgot that.

Reply:
Fixed